assembl/nlp/optics.py

```diff
@@ -1,466 +0,0 @@
-'''
```

Comment:
Was this code not functional or did it not provide much value? Otherwise what about extracting it to a new repository, in the same spirit as our other discussion analysis tools?